### PR TITLE
adding inline-blockquote functionality

### PR DESCRIPTION
**Pull request by idanizi-panw**
## Need
support for inline-code when using one back-tick like so:  \`...`

## Before
image

## After
image

## Definition of Done
- [x] Unit Tests
- [x] Manual E2E Test

### CODE REVIEW

File: src/utils/markdown.js [hunk 1]
```diff
@@ -238,3 +240,113 @@ export function mdBtn(md) {
   );
 }
 
+export function myBackticks(md) {
```
Comment (asafshen) [inline src/utils/markdown.js:243]:
```suggestion
export function mdBackticks(md) {
```

right?

**Review by asafshen — APPROVED** [review]
nice,  minor comment

@davidbinyamin can you also take a look?

File: src/utils/markdown.js (hunk 1 above)
Comment (Bars92) [reply src/utils/markdown.js:243]: where diid you get that code @idanizi-panw ? isn't it worth using existing plugins ?

Comment (idanizi-panw) [reply src/utils/markdown.js:243]: @Bars92 I took it from the `web-client`: https://code.pan.run/xsoar/web-client/-/blob/master/src/util/markdown.js#L325

**Pull request merged**